src/chat/Chat.jsx

```diff
@@ -43,14 +58,35 @@ const getDirection = (userId, classes) => {
 const Chat = () => {
     const classes = useStyles();
 
+    const [messages, setMessages] = useState(dummyMessages);
+
+    const addDummyResponse = (newMessages) => {
+        const withResponse = newMessages.concat(getDummyResponse());
+        setMessages(withResponse);
+    }
+
+    const print = (text) => {
+        if(text.length > 0) {
+            const newMessages = messages.concat({userId: '1', text: text});
+            setMessages(newMessages);
+            setTimeout(function () {
+                addDummyResponse(newMessages);
+           }, 1000);
```

Comment:
The text input needs to be cleared after a successful submit

Reply:
I fixed this in the child components state.